src/Device/Port/ConfiguredPort.cpp

```diff
@@ -99,7 +99,19 @@ OpenPortInternal(EventLoop &event_loop, Cares::Channel &cares,
     if (config.path.empty())
       throw std::runtime_error("No port path configured");
 
+#ifdef _WIN32
+    StaticString<0x20> port_path;
+    // use the windows style of device names:
+    port_path.Format(_T("\\\\.\\%s"), config.path.c_str());
+    if (!port_path.empty() && port_path.back() == TCHAR(':')) {
+      /* On Windows strip the trailing colon if the port name setting
+      has the old style 'COMx:' instead of 'COMx' */
+      port_path.Truncate(port_path.length() - 1);
+    }
```

Comment:
What's the point of copying the trailing colon to the buffer if you're going to remove it later?

Reply:
I want to strip the trealing colon, if it is still present from a previous profile setting (not used anymore in the future) - without touching the config.path.